src/Libraries/RevitNodesUI/SelectionCombo.cs

```diff
@@ -33,7 +33,7 @@ namespace Dynamo.ComboNodes
     NodeCategory(Revit.Elements.BuiltinNodeCategories.REVIT_SELECTION),
     NodeDescription("SelectModelElementsByCategoryDescription", typeof(DSRevitNodesUI.Properties.Resources)),
     IsDesignScriptCompatible]
-    public class DSModelElementByCategorySelection : ElementFilterSelection<Element>
+    public class DSModelElementsByCategorySelection : ElementFilterSelection<Element>
```

Comment:
This is an API break.

Reply:
Shouldn't break if this hasn't been released yet which I will confirm.  We don't have to update this...  It just seemed appropriate to update the class name to plural as this PR adds the actual singular Element selection version of the node.